Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -29,6 +29,9 @@ Neos:
         'trustedProxies':
           position: 'start'
           middleware: 'Neos\Flow\Http\Middleware\TrustedProxiesMiddleware'
+        'routing':
+          position: 'after trustedProxies'
```

Comment:
```suggestion
          # needs to be "after trustedProxies" because AjaxWidget comes "before routing" but still needs to be after trustedProxies
          position: 'after trustedProxies'
```